web-app/js/portal/cart/ShapeFileDownloadHandler.js

```diff
@@ -0,0 +1,65 @@
+Ext.namespace('Portal.cart');
+
+Portal.cart.ShapeFileDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
+
+    getDownloadOptions: function(filters) {
+
+        var downloadOptions = [];
+
+        if (this._showDownloadOptions(filters)) {
+
+            downloadOptions.push({
+                textKey: this.onlineResource.name,
```

Comment:
The label should come from en.js so that we get a uniform look in the drop down. 
See Example: https://github.com/aodn/aodn-portal/blob/42896f2cebb2b50b789fd207df703b46f1ea04ea/web-app/js/portal/cart/WfsDownloadHandler.js#L12

<img width="182" alt="screen shot 2017-03-15 at 2 24 21 pm" src="https://cloud.githubusercontent.com/assets/1066981/23932295/398035b6-098b-11e7-965c-bc7ae7a24554.png">